xapian-letor/tests/api_letor.cc

```diff
@@ -243,12 +239,12 @@ DEFINE_TESTCASE(preparetrainingfileonedb, generated)
     string file2;
     while (getline(if1, file1)) {
     TEST(getline(if2, file2));
-    istringstream iss1(file1);
-    istringstream iss2(file2);
+    istringstream line1(file1);
+    istringstream line2(file2);
```

Comment:
This isn't the change I proposed - it's `file1` that's poorly named here, not `iss1`.  `iss1` is an `istringstream` so the name made sense; `file1` is a string holding a line from a file, so its name is misleading.

Reading through the changes you're making, I'm becoming increasingly concerned that you're not really taking the time to understand the feedback you're being given, but are just trying to hammer out some changes to push and hope that deals with it.  The problem is that inevitably ends up in follow-up feedback and an overly long and drawn-out review process with a lot of frustration on both sides - we feel we're wasting our time trying to help as you don't seem to actually listen, while you feel that you're spending forever trying to get the patch finished.

Let's try to break that cycle.

Reply:
Sorry. I interpreted that as "A file has being read on opening the ifstream" and "A line has been read from the file" . I think I got distracted into thinking that. I'm really sorry. I will pay utmost attention onwards. 
So sorry for this.

Reply:
And what you said makes much more sense than what I interpreted. I'll fix this right away.